resources/texts/change_log.txt

```diff
@@ -1,3 +1,51 @@
+[5.9]
+
+* New features and improvements:
+- Introduced scarf seam settings in experimental, Scarf Seam Length, Scarf Seam Start Height, and Scarf Seam Split Distance
+- Updated the Start and End G-code edit options (for both Machine and Extruder) to extend to include If/Else statements and formulas
+- Introduced Build Fan Speed at Height, Build Fan Speed at Layer, and Build Volume Fan Number to control extra fans, like those controlling the build volume separately
+- Added the "Extra Infill Lines to Support Skins" setting and other improvements to help make printing over sparse infill more reliable, contributed by @Hello1024
+- Significant speed improvements interacting with custom settings especially if your printer has multiple extruders
```

Comment:
I would say "UI speed improvements" because speed improvements in this context are usually related to printing, so I got confused :laughing:

Reply:
Agreed